src/containerapp/azext_containerapp/java_decorator.py

```diff
@@ -0,0 +1,176 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from typing import Any, Dict
+
+from azure.cli.command_modules.containerapp._utils import safe_get
+from azure.cli.core.commands import AzCliCommand
+from azure.cli.core.azclierror import ValidationError, ResourceNotFoundError
+from azure.cli.command_modules.containerapp.base_resource import BaseResource
+from knack.log import get_logger
+
+from azure.cli.command_modules.containerapp._utils import _get_existing_secrets
+
+from ._models import JavaLoggerSetting
+
+from ._client_factory import handle_raw_exception
+
+logger = get_logger(__name__)
+
+
+class JavaLoggerDecorator(BaseResource):
```

Comment:
Recommend `ContainerappJavaLoggerDecorator`, because looger is part of containerapp resource. And is using ContainerappClient.